security.php

```diff
@@ -397,3 +397,22 @@ function vip_send_wplogin_header( $user ) {
 	return $user;
 }
 add_filter( 'determine_current_user', 'vip_send_wplogin_header', 10000 );
+
+/**
+ * Filters the list of disallowed usernames.
+ *
+ * @param array $usernames Array of disallowed usernames.
+ * @return array $usernames Array of disallowed usernames.
+ */
+function vip_illegal_user_logins( $logins ) {
+	if ( ! is_array( $logins ) ) {
+		$logins = [];
+	}
+
+	$logins[] = 'admin';
```

Comment:
Should we use the existing values of `wpcom_vip_is_restricted_username()`: https://github.com/Automattic/vip-go-mu-plugins/blob/a6d86cc8cb7b10f93f7835c8162068bc72615c31/security.php#L54?

Reply:
At first I wanted to, but then I realized that the WPCOM_VIP_MACHINE_USER_LOGIN useris created automatically and, therefore, is not a forbidden name. Which leaves us only admin.